database/PracticeOperation/0005_DataGenerateScript.sql

```diff
@@ -0,0 +1,68 @@
+USE [PatientCheckIn];
+
+SET TRANSACTION ISOLATION LEVEL READ UNCOMMITTED;
+SET QUOTED_IDENTIFIER ON;
+SET NOCOUNT ON;
+SET ANSI_NULLS ON;
+
+GO
+
+INSERT INTO dbo.Patient (PatientIdentifier, FirstName, MiddleName, LastName, FullName, DoB, 
```

Comment:
Should 
```
INSERT INTO TABLE
VALUES(...)
VALUES(...)
VALUES(...)
```

Reply:
This has been applied in a appropriate places. Please review this @vietvo361